images/manageiq-orchestrator/container-assets/entrypoint

```diff
@@ -16,6 +16,33 @@ function check_svc_status() {
   echo "${SVC_NAME}:${SVC_PORT} - accepting connections"
 }
 
+function update_auth_settings() {
+  echo "== Updating auth settings =="
+  script=$(cat <<- EOS
+    provider_type = case ENV['AUTH_TYPE']
+                    when 'saml'
+                      'saml'
+                    when 'openid-connect'
+                      'oidc'
+                    else
+                      'none'
+                    end
+    settings = {
+      'authentication/mode'                 => ENV['AUTH_TYPE'] != "internal" ? 'httpd' : 'database',
+      'authentication/httpd_role'           => ENV['AUTH_TYPE'] != "internal" ? true : false,
+      'authentication/sso_enabled'          => ENV['AUTH_SSO'] == "true",
+      'authentication/saml_enabled'         => provider_type == 'saml',
+      'authentication/oidc_enabled'         => provider_type == 'oidc',
+      'authentication/provider_type'        => provider_type,
+      'authentication/local_login_disabled' => ENV['LOCAL_LOGIN_DISABLED'] == "true"
+    }
+    MiqRegion.my_region.add_settings_for_resource(settings)
+EOS
+)
+
+  bin/rails runner "$script"
```

Comment:
Not for this PR, but I'm wondering if this should be a core rake task.  Then here we can simplify this to something like `bundle exec rake evm:auth:setup`

Reply:
Yeah, there was the one for setting up settings, but creating the command line for that was going to be just as difficult as this. Plus it only worked for server settings and we needed region here.